**Core -> Full Integration (with SecurityManager)** [Build 155](https://ci.wildfly.org/buildConfiguration/WildFlyCore_PullRequest_WildFlyCoreFullIntegrationLinuxJdk25/565423) outcome was **FAILURE** using a merge of ce4b8770d81bc0c105f796f9aaa2e38a8b8fc4b1
Summary: Tests failed: 1 (1 new), passed: 8109, ignored: 139 Build time: 05:52:06
### Failed tests
```
org.wildfly.test.integration.microprofile.jwt.propagation.JWTIdentityPropagationTestCase(standalone-enabled-expansion-test).testInvokeEJBWithinSingleDeploymentOutflowAnonymousConfigured: java.lang.AssertionError: Call was authenticated: expectedMessage:[targetCallerPrincipal: anonymous, targetIsCallerAdmin: false] returned msg:[hello testUser  hasJWT: true  targetCallerPrincipal: testUser, targetIsCallerAdmin: false  ctx: org.jboss.resteasy.plugins.server.servlet.ServletSecurityContext@57850505  targetBean context: org.jboss.as.ejb3.context.SessionContextImpl@7558a112]
	at org.wildfly.test.integration.microprofile.jwt.propagation.JWTIdentityPropagationTestCase.testInvokeEJB(JWTIdentityPropagationTestCase.java:323)
	at org.wildfly.test.integration.microprofile.jwt.propagation.JWTIdentityPropagationTestCase.testInvokeEJBWithinSingleDeploymentOutflowAnonymousConfigured(JWTIdentityPropagationTestCase.java:271)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
------- Stdout: -------
 [31m04:58:38,564 ERROR [stderr] (default task-2) java.lang.Throwable: hello testUser  hasJWT: true  targetCallerPrincipal: testUser, targetIsCallerAdmin: false  ctx: org.jboss.resteasy.plugins.server.servlet.ServletSecurityContext@57850505  targetBean context: org.jboss.as.ejb3.context.SessionContextImpl@7558a112
 [0m [31m04:58:38,565 ERROR [stderr] (default task-2) 	at deployment.outflow-anonymous-config.ear.outflow-anonymous-config-web.war//org.wildfly.test.integration.microprofile.jwt.propagation.BeanEndPoint.helloRolesAllowed(BeanEndPoint.java:50)
 [0m [31m04:58:38,565 ERROR [stderr] (default task-2) 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
 [0m [31m04:58:38,565 ERROR [stderr] (default task-2) 	at org.jboss.as.ee@41.0.0.Beta1-SNAPSHOT//org.jboss.as.ee.component.ManagedReferenceMethodInterceptor.processInvocation(ManagedReferenceMethodInterceptor.java:35)
 [0m [31m04:58:38,565 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)
 [0m [31m04:58:38,565 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext$Invocation.proceed(InterceptorContext.java:509)
 [0m [31m04:58:38,565 ERROR [stderr] (default task-2) 	at org.jboss.as.weld.common@41.0.0.Beta1-SNAPSHOT//org.jboss.as.weld.interceptors.Jsr299BindingsInterceptor.delegateInterception(Jsr299BindingsInterceptor.java:62)
 [0m [31m04:58:38,566 ERROR [stderr] (default task-2) 	at org.jboss.as.weld.common@41.0.0.Beta1-SNAPSHOT//org.jboss.as.weld.interceptors.Jsr299BindingsInterceptor.doMethodInterception(Jsr299BindingsInterceptor.java:72)
 [0m [31m04:58:38,566 ERROR [stderr] (default task-2) 	at org.jboss.as.weld.common@41.0.0.Beta1-SNAPSHOT//org.jboss.as.weld.interceptors.Jsr299BindingsInterceptor.processInvocation(Jsr299BindingsInterceptor.java:85)
 [0m [31m04:58:38,566 ERROR [stderr] (default task-2) 	at org.jboss.as.ee@41.0.0.Beta1-SNAPSHOT//org.jboss.as.ee.component.interceptors.UserInterceptorFactory$1.processInvocation(UserInterceptorFactory.java:46)
 [0m [31m04:58:38,566 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)
 [0m [31m04:58:38,566 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@41.0.0.Beta1-SNAPSHOT//org.jboss.as.ejb3.component.invocationmetrics.ExecutionTimeInterceptor.processInvocation(ExecutionTimeInterceptor.java:26)
 [0m [31m04:58:38,566 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)
 [0m [31m04:58:38,566 ERROR [stderr] (default task-2) 	at org.jboss.as.jpa@41.0.0.Beta1-SNAPSHOT//org.jboss.as.jpa.interceptor.SBInvocationInterceptor.processInvocation(SBInvocationInterceptor.java:30)
 [0m [31m04:58:38,566 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)
 [0m [31m04:58:38,566 ERROR [stderr] (default task-2) 	at org.wildfly.concurrency@41.0.0.Beta1-SNAPSHOT//org.jboss.as.ee.concurrent.ConcurrentContextInterceptor.processInvocation(ConcurrentContextInterceptor.java:28)
 [0m [31m04:58:38,566 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)
 [0m [31m04:58:38,566 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InitialInterceptor.processInvocation(InitialInterceptor.java:40)
 [0m [31m04:58:38,567 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)
 [0m [31m04:58:38,567 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.ChainedInterceptor.processInvocation(ChainedInterceptor.java:53)
 [0m [31m04:58:38,567 ERROR [stderr] (default task-2) 	at org.jboss.as.ee@41.0.0.Beta1-SNAPSHOT//org.jboss.as.ee.component.interceptors.ComponentDispatcherInterceptor.processInvocation(ComponentDispatcherInterceptor.java:35)
 [0m [31m04:58:38,567 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)
 [0m [31m04:58:38,567 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@41.0.0.Beta1-SNAPSHOT//org.jboss.as.ejb3.component.pool.PooledInstanceInterceptor.processInvocation(PooledInstanceInterceptor.java:34)
 [0m [31m04:58:38,567 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)
 [0m [31m04:58:38,567 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@41.0.0.Beta1-SNAPSHOT//org.jboss.as.ejb3.component.interceptors.AdditionalSetupInterceptor.processInvocation(AdditionalSetupInterceptor.java:39)
 [0m [31m04:58:38,567 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)
 [0m [31m04:58:38,567 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@41.0.0.Beta1-SNAPSHOT//org.jboss.as.ejb3.tx.CMTTxInterceptor.invokeInOurTx(CMTTxInterceptor.java:237)
 [0m [31m04:58:38,567 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@41.0.0.Beta1-SNAPSHOT//org.jboss.as.ejb3.tx.CMTTxInterceptor.required(CMTTxInterceptor.java:373)
 [0m [31m04:58:38,567 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@41.0.0.Beta1-SNAPSHOT//org.jboss.as.ejb3.tx.CMTTxInterceptor.processInvocation(CMTTxInterceptor.java:143)
 [0m [31m04:58:38,568 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)
 [0m [31m04:58:38,568 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext$Invocation.proceed(InterceptorContext.java:509)
 [0m [31m04:58:38,568 ERROR [stderr] (default task-2) 	at org.jboss.weld.core@6.0.4.Final//org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:78)
 [0m [31m04:58:38,568 ERROR [stderr] (default task-2) 	at org.jboss.as.weld.common@41.0.0.Beta1-SNAPSHOT//org.jboss.as.weld.ejb.EjbRequestScopeActivationInterceptor.processInvocation(EjbRequestScopeActivationInterceptor.java:72)
 [0m [31m04:58:38,568 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)
 [0m [31m04:58:38,568 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@41.0.0.Beta1-SNAPSHOT//org.jboss.as.ejb3.component.interceptors.CurrentInvocationContextInterceptor.processInvocation(CurrentInvocationContextInterceptor.java:24)


```

--- a/pom.xml
+++ b/pom.xml
@@ -233,7 +233,7 @@
         <version.com.jcraft.jsch>0.1.55</version.com.jcraft.jsch>
         <version.commons-cli>1.11.0</version.commons-cli>
         <version.commons-collections>3.2.2</version.commons-collections>
-        <version.commons-daemon>1.5.1</version.commons-daemon>
+        <version.commons-daemon>1.6.0</version.commons-daemon>
         <version.commons-lang3>3.20.0</version.commons-lang3>
         <version.commons-io>2.21.0</version.commons-io>
         <version.io.netty>4.1.134.Final</version.io.netty>

